frontend/jsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I am absolutely not sure what this file is about ? Perhaps we need it but if you know can you please elaborate it here, Thanks

Reply:
I also never used this, I think this is vite specific(?) config allowing shorter paths in imports with `@` - but I would like to know more.